server/views/licenceConditions/includes/appointmentName.pug

```diff
@@ -6,13 +6,13 @@ div.pure-g.paddingBottom
 
 div.inline
   div.multiple-choice
-    input(id='psychiatrist' type='radio' name='appointmentProfession' value='psychiatrist' checked=condition.user_submission && condition.user_submission.appointmentProfession === 'Psychiatrist')
+    input(id='psychiatrist' type='radio' name='appointmentProfession' value='psychiatrist' checked=condition.user_submission && condition.user_submission.appointmentProfession === 'psychiatrist')
     label(for='psychiatrist') Psychiatrist
 
   div.multiple-choice
-    input(id='psychologist' type='radio' name='appointmentProfession' value='psychologist' checked=condition.user_submission && condition.user_submission.appointmentProfession === 'Psychologist')
+    input(id='psychologist' type='radio' name='appointmentProfession' value='psychologist' checked=condition.user_submission && condition.user_submission.appointmentProfession === 'psychologist')
     label(for='psychologist') Psychologist
 
   div.multiple-choice
-    input(id='medicalPractitioner' type='radio' name='appointmentProfession' value='medical practitioner' checked=condition.user_submission && condition.user_submission.appointmentProfession === 'Medical practitioner')
+    input(id='medicalPractitioner' type='radio' name='appointmentProfession' value='medical practitioner' checked=condition.user_submission && condition.user_submission.appointmentProfession === 'medical practitioner')
     label(for='medicalPractitioner') Medical practitioner
```

Comment:
Looks as if this part of the form has no test coverage.  Perhaps we need to add some tests.   I think this is outside the scope of this story/bug.  Consider raising a separate ticket for this.

Reply:
ok